sdk/clientcore/core/src/main/java/io/clientcore/core/http/models/Response.java

```diff
@@ -83,7 +83,7 @@ public void close() {
             try {
                 ((Closeable) value).close();
             } catch (IOException e) {
-                throw LOGGER.logThrowableAsError(CoreException.from(e, false));
+                throw LOGGER.throwableAtError().log((msg, cause) -> CoreException.from(msg, e, false));
```

Comment:
Where exactly does `cause` come from here again? Should we be consuming it?

Reply:
it's a typo, it should be `LOGGER.throwableAtError().log(e, (msg, cause) -> CoreException.from(msg, cause, false)), and `cause` is `e`